Stripe/STPRedirectContext.m

```diff
@@ -19,6 +20,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef void (^STPNativeRedirectCompletionBlock)(BOOL success);
```

Comment:
nit: can we call this `STPBoolCompletionBlock` to match our other block type names?

Reply:
Hmm, should i move it to STPBlocks.h ? I wasn't sure about just naming it just BoolBlock because the bool is specifically success or failure and wasn't sure if that was generic across what all bool using blocks would be.